content/git/concepts/status/status.md

```diff
@@ -16,18 +16,19 @@ In Git, the `status` command shows the state of the working directory, staging a
 
 ## Example
 
-Below is a short example of how the `git status` command works. Assuming a change was saved in a text file, **example.txt**, running `git status` will show that:
+Below is a short example of how the `git status` command works. Running `git status` will show the below, assuming a change was saved in a text file **example.txt** (but not added to the staging area), and **README.txt** is a new file that was added to the staging area:
 
 ```shell
 $ git status
```

Comment:
$ git status

On branch main

No commits yet
Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
  new file: README.txt

Untracked files:
  (use "git add <file>..." to include in what will be committed)
  example.txt

nothing added to commit but untracked files present (use "git add" to track)